tests/snapshots/assets__check_file@cetz-manual.typ-40.snap

```diff
@@ -336,7 +336,8 @@ line(..c)
     "array",
     "dictionary",
   ),
-  [This option sets the mark to draw when using the `mark` draw function, or applies styling to both mark ends of path based elements. The mark's name or shorthand can be given, multiple marks can be drawn by passing an array of names or shorthands. When `none` no marks will be drawn. A style `dictionary` can be given instead of a `string` to override styling for that particular mark, just make sure to still give the mark name using the `symbol` key otherwise nothing will be drawn!. ],
+  [This option sets the mark to draw when using the `mark` draw function, or applies styling to both mark ends of path based elements. The mark's name or shorthand can be given, multiple marks can be drawn by passing an array of names or shorthands. When `none` no marks will be drawn. A style `dictionary` can be given instead of a `string` to override styling for that particular mark, just make sure to still give the mark name using the `symbol` key otherwise nothing will be drawn!.
+  ],
 )
```

Comment:
Only the right bracket is pushed into next line. This might not be intended? Perhaps we should keep it if only one side has space

Reply:
I think the writer is to blame for this extra space

Reply:
We should reach a consensus on this.
If the extra space is a mistake, breaking the line can help the write notice the mistake.
I feel it like a strange flavor when choosing to keep it with one-side space.